README.md

```diff
@@ -1 +1,5 @@
-Use the command `colcon build` to build the project.
\ No newline at end of file
+Use the command `colcon build` to build the project.
+
+The following is a diagram of how the software architecture communicates at a high level:
+
+![Software Architecture Flowchart](images/MicroBullProposalFlowchart_rev1.png)
```

Comment:
We don't have an encoder as a hardware. Is this a software feedback?

Reply:
I thought that we were 3D printing an encoder? If that plan has changed then that step might need to be removed. Although it would still be nice to have a sensor telling us how much we have moved. Otherwise we would be solely relying on the ultrasonic distance sensors.